nhais/outbound/request/handler.py

```diff
@@ -23,9 +27,52 @@ def __init__(self, application: "Application", request: httputil.HTTPServerReque
             password=config.get_config('OUTBOUND_QUEUE_PASSWORD', default=None))
         self.fhir_to_edifact = FhirToEdifact()
         self.mesh_wrapper = MeshOutboundWrapper(queue_adaptor)
+        root_dir = os.path.dirname(os.path.abspath(__file__))
+        self.json_schema_patient = root_dir + "/json-schema-patient.json"
+
+    def validate_uri_matches_payload_operation(self, operation, uri):
+        string_length_of_operation = uri.find(operation)
+        if (string_length_of_operation >= len(operation) - 1):
```

Comment:
Why not just return the comparison result?

Reply:
It was for readability but I have amended this